R/module_teal_with_splash.R

```diff
@@ -1,17 +1,18 @@
 # This file adds a splash screen for delayed data loading on top of teal
 
-#' UI to show a splash screen in the beginning, then delegate to [srv_teal()]
```

Comment:
is ticks required?
(there are more examples like this)

Reply:
As part of our pre-release, we have added backticks in all places where they were inconsistent in the documentation.

Reply:
Usually I saw just `[foo()]` and not ``[`foo()`]``. That's also in line with roxygen2 manyal - see example [here](https://roxygen2.r-lib.org/articles/index-crossref.html#see-also). Did you check if roxygen does not complain?

Reply:
We did, it interprets both the same.

Reply:
OK so then let's leave it. It doesn't follow the standards but that's fine as long as roxygen does not complain.